index.html

```diff
@@ -9,11 +9,71 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <!-- header starts here-->
+<header id="header">  
+   <nav id="nav-bar"> 
+     <img src= "img\karma-logo.svg" alt="Karma">
+     <ul class="nav">
+       <li> <a class="nav-link" href="#MeetKarma"> Meet Karma </a></li>
+       <li> <a class="nav-link" href="#Howitworks"> How it works </a> </li>   
+       <li> <a class="nav-link" href="#Store"> Store </a> </li> 
+       <li> <a class="nav-link" href="#Blog"> Blog </a> </li>
+       <li> <a class="nav-link" href="#Help"> Help </a></li>
+       <li> <a class="nav-link" href="#Login"> Login</a></li> 
+     </ul>
+  </nav> 
+</header> <!--header ends here-->
+
+<article>
```

Comment:
```suggestion
<main>
```
This seems like it should be a main, since we normally have `header` `main` `footer` as the top level elements on a page